components/Newsletter.tsx

```diff
@@ -0,0 +1,38 @@
+
+
+const Newsletter = () => (
+    
+        <div className="container">
+            <section className="section mt-4">
+                <div className="columns is-centered is-vcentered">
+                    <div className="column is-half">
+                        <h1 className="title is-1 is-spaced has-text-centered mt-4" ><span style={{whiteSpace: "nowrap"}}>Join our </span> <span style={{whiteSpace: "nowrap"}}>Newsletter</span></h1>
+                        <div className="field">
+                            <label className="label">First name</label>
+                            <div className="control">
+                                <input className="input" type="text" id="fname" name="fname"></input>
+                            </div>
+                        </div>
+                        <div className="field">
+                            <label className="label">Last name</label>
+                            <div className="control">
+                                <input className="input" type="text" id="lname" name="lname"></input>
+                            </div>
+                        </div>
+                        <div className="field">
+                            <label className="label">UTD email</label>
+                            <div className="control">
+                                <input className="input" type="text" placeholder="abc123456@utdallas.edu" id="email" name="email"></input>
+                            </div>
+                        </div>
+                        <div className="control">
+                            <button className="button is-primary">Submit</button>
+                        </div>
+                    </div>
+                </div>
+            </section>
+        </div>
+  
+  )
+
+  export default Newsletter
```

Comment:
It'd be great to add Eddie's new changes to the newsletter - I'll refactor the current newsletter page into a component like you have here and have the newsletter page use that component. I'd prefer to keep the newsletter component out of this PR (in terms of adding the component and updating the newsletter page), I'll do that and request a review from you on that PR

Reply:
Actually if @techmin wants to work on that, that would be great! Looks like Atmin created the Newsletter component originally. I'll bring this up on Saturday and see if he's interested in working on that PR.

Reply:
This is no longer valid since we're going to be putting social media there

Reply:
oh yeah we were on call when we did this so i'll take the blame for shoving too much into one PR. 

I removed the Newsletter component, merged with the InstagramFeed branch, and added that component to the homepage

Reply:
There seems to be a problem with the Instagram component though. It does not load for me (even in the instagram-feed branch). Not sure if the problem is local.